script.js

```diff
@@ -1,4 +1,221 @@
+          "<br>" +
+          "You won the last round and therefore you win muk-jji-ppa!";
+      } else if (lastroundWinner == "comp") {
+        lastroundWinner = "";
+        myOutputValue =
+          drawOutcome +
+          "<br>" +
+          "You lost the last round and therefore you have lost muk-jji-ppa.";
+      } else {
+        myOutputValue =
+          drawOutcome +
+          "<br>" +
+          "No one won the previous round, so no one is declared the winner.";
+      }
+    } else {
+      myOutputValue =
+        "Invalid input! Please only enter either rock, paper or scissors(small caps).";
+    }
+  }
+  if (currentMode == "compkoreanRPS") {
+    var playercompChoice = RandRoll();
+    var outcome =
+      `You picked ${playercompChoice}.` +
+      "<br>" +
+      `The computer chose ${compChoice}`;
+    var winOutcome =
+      outcome + "<br>" + `You won this round, ${username}! Play again?`;
+    var loseOutcome =
+      outcome + "<br>" + `You lost this round, ${username}! Play again?`;
+    var drawOutcome = outcome + "<br>" + `You drew, ${username}! Play again?`;
+    if (
+      (playercompChoice == "scissors" && compChoice == "paper") ||
+      (playercompChoice == "paper" && compChoice == "rock") ||
+      (playercompChoice == "rock" && compChoice == "scissors")
+    ) {
+      lastroundWinner = "player";
+      myOutputValue =
+        winOutcome +
+        "<br>" +
+        "You need to draw the next round in order to be declared the winner of muk-jji-ppa!";
+    } else if (
+      (playercompChoice == "scissors" && compChoice == "rock") ||
+      (playercompChoice == "paper" && compChoice == "scissors") ||
+      (playercompChoice == "rock" && compChoice == "paper")
+    ) {
+      lastroundWinner = "comp";
+      myOutputValue =
+        loseOutcome +
+        "<br>" +
+        "The computer will muk-jji-ppa if it is a draw next round!";
+    } else if (playercompChoice == compChoice) {
+      if (lastroundWinner == "player") {
+        lastroundWinner = "";
+        myOutputValue =
+          drawOutcome +
+          "<br>" +
+          "You won the last round and therefore you win muk-jji-ppa!";
+      } else if (lastroundWinner == "comp") {
+        lastroundWinner = "";
+        myOutputValue =
+          drawOutcome +
+          "<br>" +
+          "You lost the last round and therefore you have lost muk-jji-ppa.";
+      } else {
+        myOutputValue =
+          drawOutcome +
+          "<br>" +
+          "No one won the previous round, so no one is declared the winner.";
+      }
+    }
+  }
   return myOutputValue;
 };
+
+var RandRoll = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var RandNumber = randomInteger + 1;
+  console.log(RandNumber);
+  //1 is paper, 2 is rock, and 3 is scissors
+  if (RandNumber == 1) {
+    return "paper";
+  } else if (RandNumber == 2) {
+    return "rock";
+  } else {
+    return "scissors";
+  }
+};
```

Comment:
good job pulling this out as a helper function:) hope to see more of this for the other logic intensive code in your main function.